crates/secrets-nats-kv/src/main.rs

```diff
@@ -107,6 +110,35 @@ struct PutCommand {
     global: GlobalOpts,
 }
 
+#[derive(Parser, Debug, Clone, ValueEnum)]
+enum SecretKind {
+    String,
+    Binary,
+}
+
+#[derive(Parser, Debug, Clone)]
+struct GetCommand {
+    /// The server's encryption XKey to decrypt the secret fetched from the KV store.
+    #[clap(short, long, env = "ENCRYPTION_XKEY_SEED")]
+    encryption_xkey_seed: String,
+    /// The NATS address to connect to where the backend is running
+    #[clap(long, default_value = "127.0.0.1:4222")]
+    nats_address: String,
```

Comment:
NIT: some of these (like `nats_address` in particular) MIGHT be better in `GlobalOpts`, but that works as a follow up!

Reply:
Agreed! Good opportunities to clean that project up